app/styles/globalPromotions.scss

```diff
@@ -4,10 +4,26 @@
     justify-content: space-between;
     align-items: center;
   }
-}
 
-.status-label {
-  display: inline-block;
-  min-width: 70px;
-  vertical-align: middle;
+  .button-container {
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+    margin-top: 25px;
```

Comment:
I don't love the hard-coded margin to make the button aligned with the bottom of the filters. Bootstrap doesn't seem to make this easy either. I wonder if rewriting the filters row as flexbox instead of using Bootstrap's rows and columns could achieve this easier?

However, I don't want to stall this, so feel free to merge and move onto other things.